sound/soc/codecs/rt715-sdca.c

```diff
@@ -286,7 +288,22 @@ static int rt715_sdca_put_volsw(struct snd_kcontrol *kcontrol,
 		if (err < 0)
 			return err;
 	}
-
+#if IS_ENABLED(CONFIG_DELL_PRIVACY)
```

Comment:
@bardliao I think the confusing thing is there are other commits that will not come through sof tree.  Should they be included in the PR for review purpose only?

Reply:
> @bardliao I think the confusing thing is there are other commits that will not come through sof tree. Should they be included in the PR for review purpose only?

Yes, it would be better.